core/core.ts

```diff
@@ -128,6 +128,10 @@ export function stringValue(value: string): ResourceValue {
   return { kind: 'text', text: quotedText(value) };
 }
 
+export function quotedStringValue(value: string): ResourceValue {
```

Comment:
Could you please add a comment to this function explaining why a user may need to call this as opposed to `stringValue` include details about the newline thing

Reply:
Added